### PR TITLE
Expose bridge metrics

### PR DESCRIPTION
**Pull request by larshesel**
So far we only have a metric for dropped messages due to the queue
being full.

### CODE REVIEW

File: apps/vmq_bridge/priv/vmq_bridge.schema [hunk 1]
```diff
@@ -265,6 +266,21 @@
          Term
  end}.
 
+{mapping, "vmq_bridge.vmq_metrics_mfa", "vmq_bridge.vmq_metrics_mfa",
```
Comment (dergraf) [inline apps/vmq_bridge/priv/vmq_bridge.schema]: not sure if we really need this to be in the schema...thoughts?

File: apps/vmq_commons/src/gen_mqtt_client.erl [hunk 2]
```diff
@@ -161,6 +164,10 @@ start(Name, Module, Args, Opts) ->
 info(Pid) ->
     gen_fsm:sync_send_all_state_event(Pid, info).
 
+stats(Pid) ->
+    [{dictionary, PD}] = erlang:process_info(Pid, [dictionary]),
```
Comment (dergraf) [inline apps/vmq_commons/src/gen_mqtt_client.erl]: :1st_place_medal:

Comment (larshesel) [reply apps/vmq_commons/src/gen_mqtt_client.erl]: You like that, huh?

File: apps/vmq_bridge/priv/vmq_bridge.schema (hunk 1 above)
Comment (larshesel) [reply apps/vmq_bridge/priv/vmq_bridge.schema]: adding it the app.src file only isn't enough(I thought it was?) - it's anyway hidden and won't be in the config file.

Comment (dergraf) [reply apps/vmq_bridge/priv/vmq_bridge.schema]: to my knowledge app.src is sufficient... I tested it with my SWC release.

Comment (larshesel) [reply apps/vmq_bridge/priv/vmq_bridge.schema]: You're right - I'd done something wrong when testing. Removed it.

File: changelog.md [hunk 3]
```diff
@@ -46,7 +46,7 @@
   reusing already available data.
 - Enable plugins to expose metrics via the available metric providers.
 - Add histogram metric type.
-- Fix incorrect debug log message in QoS 2 flow.
```
[the remark this answers is not in the document]
Comment (dergraf) [reply changelog.md]: Otherwise good to be merged

[the remark this answers is not in the document]
Comment (larshesel) [reply changelog.md]: oops, think I removed the wrong line there when rebasing... good catch

[the remark this answers is not in the document]
Comment (larshesel) [reply changelog.md]: fixed.

**Pull request merged**